ipa-core/src/query/runner/hybrid.rs

```diff
@@ -305,30 +315,26 @@ mod tests {
 
             // TODO: after landing #1446, refactor this to only take the first 3
```

Comment:
does this PR address this TODO or we need to do something extra?

Reply:
good call, this can go!